scripts/oecd/regional_demography/life_expectancy/manifest.json

```diff
@@ -0,0 +1,22 @@
+{
+  "import_specifications": [
+    {
+      "import_name": "OECDRegionalDemography_LifeExpectancy",
+      "curator_emails": [
+        "chharish@google.com"
+      ],
+      "provenance_url": "https://data-explorer.oecd.org/vis?df[ds]=DisseminateFinalDMZ&df[id]=DSD_REG_DEMO%40DF_LIFE_EXP&df[ag]=OECD.CFE.EDS&df[vs]=2.0&dq=A.......&pd=%2C&to[TIME_PERIOD]=false&vw=tb",
+      "provenance_description": "Life Expectancy statistics by gender measured by OECD (Organisation for Economic Co-operation and Development).",
+      "scripts": [
+        "preprocess_csv.py"
+      ],
+      "import_inputs": [
+        {
+          "template_mcf": "OECD_life_expectancy_cleaned.tmcf",
+          "cleaned_csv": "OECD_life_expectancy_cleaned.csv"
+        }
+      ],
+      "cron_schedule": "0 0 1 3 *"
```

Comment:
Please update different cron_schedule for each import.

Reply:
Gave a 3hrs gap between each schedule. Correct me if wrong.